services/modules/slack/manager.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import unicode_literals
+import random
+import string
+import requests
+from eveonline.managers import EveManager
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from slackclient import SlackClient
+
+from six import iteritems
+from .tasks import SlackTasks
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SlackManager:
+    def __init__(self):
+        pass
+
+    @classmethod
+    def _response_ok(cls, response):
+        try:
+            if response['ok']:
+                return True
+            else:
+                return False
+        except:
+            return False
+
+    @staticmethod
+    def exec_request(endpoint, func, email, **kwargs):
```

Comment:
I would get rid of this function entirely and drop the static manager class.

So instead you get
```
class SlackManager:
    def __init__(self):
        self.client = SlackClient(settings.SLACK_TOKEN)
...
    def invite_user(self, user):
        logger.debug("Inviting to Slack with email %s" % user.email)
        ret = self.client.api_call('users.admin.invite', email=email, resend=True)
...
```